src/controllers/admin.ts

```diff
@@ -0,0 +1,73 @@
+import * as Hapi from 'hapi'
+import Config from '../services/Config'
+import PodDatabase from '../services/PodDatabase'
+import { Injector } from 'reduct'
+
+import { create as createLogger } from '../common/log'
+const log = createLogger('admin')
+
+export default function (server: Hapi.Server, deps: Injector) {
+  const podDatabase = deps(PodDatabase)
+  const config = deps(Config)
+
+  function getCurrencyPerSecond (): number {
```

Comment:
This is already in `pods.ts`, so to keep it DRY I think we should move into a util and import it